### ISSUE TITLE
FR: Allow to expose port for HA Add-On Version direct UI access

### ISSUE THREAD

**Issue by raihei** [matt8707/ha-fusion#171]
Hi matt8707,

really a great project, thank you very much!

If possible it would be great to have the option to expose the add-on version container on a port, so the ha-fusion gui can be accessed without having the home-assistant gui around it. For example, for kiosk mode use on a tablet.

Or is there a way to do it, that I am not aware off?

regards
raihei

**Comment by matt8707** [comment]
https://github.com/matt8707/ha-fusion/releases/tag/2024.1.6

**Comment by SilvanLaroo** [comment]
Awesome work! I tried the latest version, and I can access it via the specified port now. However, it seems it cannot connect to Home Assistant; I get an error: ERR_HASS_HOST_REQUIRED.

**Comment by matt8707** [comment]
1. Open ingress, this registers hassUrl
2. Profit

**Comment by jpleite** [comment]
hi @matt8707 I'm not able to access via URL. I can access my HA via local_url:8124 and via my domain. If i access  local_url:8099 i get a ERR_CONNECTION_REFUSED. Any hints?

**Comment by raihei** [comment]
@matt8707 Thank you for the really fast implementation of this feature!

@jpleite Did you enable the port in the configuration Tab of the Add-on and entered the port number?

**Comment by matt8707** [comment]
Yeah, discussion https://github.com/matt8707/ha-fusion/discussions/222
